### ISSUE TITLE
Интеграция: координаты фото, RetroMap, API, OSM / Integration: geo coord., retromap.ru, API, OSM what is here

### ISSUE THREAD

**Issue by mkgrgis** [PastVu/pastvu#458]
Feature request

Рядом с кнопками 
![изображение](https://user-images.githubusercontent.com/41448637/152724476-124c6837-71a7-4dcd-b4ec-5d57bf1ca73e.png)
добавить:
1. Кнопку "скопировать координаты" в формате аналогичном координатной пипетке Retromap.ru. Некоторые идеи есть на https://pastvu.com/news/141?hl=comment-2021370 (при наличии координат)
2. Кнопку-ссылку "это место на retromap.ru" (при наличии координат)
3. Кнопку-ссылку "Скачать данные описания" - ссылка на JSON API
4. Кнопку-ссылку "Что здесь на OSM" https://www.openstreetmap.org/query?lat=...&lon=... (при наличии координат)

Думаю, что все измнения связаны с https://github.com/PastVu/pastvu/blob/master/public/js/module/photo/photo.js

Add buttons to photo's page:
1. Copy coordinates
2. Here by retromap.ru
3. Download data (API)
4. What is here on OSM

**Comment by kabalin** [comment]
3 и 4 (и может быть даже 1) мне кажется большинству людей не нужно. Можно сделать настройку профиля "Показывать геоданные", в этом случае добавлять кнопки к снимку.

**Comment by kabalin** [comment]
А как ссылка на ретромап должна выглядеть - есть какой-то фиксированный формат запроса?

**Comment by mkgrgis** [comment]
Да, формат постоянный уже несколкьо лет. [http://retromap.ru/z14_59.693651,30.461139](http://retromap.ru/z14_59.693651,30.461139) - выводит на OSM Carto , а сверху линейка доступных исторических карт. Как раз то самое что нужно пользователям.
Кнопка №4 может быть интересна для выяснения ближайших объектов или другого деления, кроме административного. Иногда для иностранных изображений сложно понять что и где. А так сразу регионы PastVu сравниваешь с глубиной деления на OSM и прикидываешь что да где.
Кнопка №3 должна первично популяризовать работу с API, обозначить присутствие возможностей для тех, кто мало задумывается и, может быть, не знает JSON.

**Comment by mkgrgis** [comment]
> Можно сделать настройку профиля "Показывать геоданные" (чекбокс), в случае если установлена, добавлять кнопки к снимку.

Не могу ручаться за все способы просмотра сайта, но рядом с существующими кнопками, по моему мнению, вплоне достаточно места для 4 безусловно показываемых таких же кнопок. Есть ли смысл их отключать или включать настройкой? Не думаю, что такая настрйока будет очень востребована.

**Comment by paul-k-pastvu** [comment]
П. 1 уже предлагался в #118

**Comment by mkgrgis** [comment]
@paul-k-pastvu , по эргономике https://github.com/PastVu/pastvu/issues/118 вроде бы лучше проработано. Тогда три кнопки логично унести в модуль карты, который пока что отделяется ув. Русланом @kabalin в черновом режиме.
На днях попробую новый docker на который прешли, есть шанс запустить отладку.
